application/src/main/resources/ecc.yml

```diff
@@ -27,16 +27,23 @@ connection:
     host: localhost
     port: 9042
     ##
-    ## Connection Timeout for CQL.
+    ## Connection Timeout for an CQL attempt.
     ## Specify a time to wait for cassandra to come up.
-    ## Connection is tried every five second until either the timeout time passes or the connection is successful.
+    ## Connection is tried based on retry policy delay calculations. Each connection attempt will use timeout to calculate CQL Connection proccess delay.
     ##
     timeout:
       time: 60
       unit: seconds
     retryPolicy:
+      ## Max number of attempts that ecChronos will try to connect with Cassandra.
       maxAttempts: 5
+      unit: seconds
```

Comment:
I would move this setting below maxDelay instead, since it is how the style is in the rest of the yml file. It is more logical to have after the settings it is related to.